jf_agent/__init__.py

```diff
@@ -46,7 +46,7 @@ def write_file(outdir, filename_prefix, compress, results):
             outfile.write(json.dumps(results, indent=2, default=str).encode('utf-8'))
     else:
         with open(f'{outdir}/{filename_prefix}.json', 'w') as outfile:
-            outfile.write(json.dumps(results, indent=2, default=str))
+            outfile.write(json.dumps(results, indent=2, default=str))   
```

Comment:
spurious diff? I think there's a setting in PyCharm to remove trailing whitespace from each line